src/components/orchestrator/Orchestrator.test.tsx

```diff
@@ -52,13 +52,16 @@ describe('Orchestrator', () => {
   const queryClient = new QueryClient()
   const OrchestratorTestWrapper = ({
     mode,
+    isDownloadEnabled,
```

Comment:
```suggestion
    isDownloadEnabled = false,
```

This will allow you to remove the various `isDownloadEnabled={false}` that seem redundant below.

It will make the tests way clearer.